### DISCUSSION TITLE
Cluster Question: Can "rails: secret_key:" config setting be the same across nodes in a Postal Cluster?

### DISCUSSION THREAD

**Discussion by JohnHammell** [postalserver/postal#3200] [Help with using Postal] [answered]
Can "rails: secret_key:" config setting be the same across nodes in a Postal Cluster?

**Comment by willpower232** [comment, marked as answer]
If all the nodes are using the same database then it should definitely be kept the same throughout otherwise I imagine you would get some weird behaviour.
